x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-TopAggregator.java.st

```diff
@@ -100,7 +103,13 @@ $endif$
         private final $Name$BucketedSort sort;
 
         private GroupingState(BigArrays bigArrays, int limit, boolean ascending) {
+$if(BytesRef)$
+            // TODO pass the breaker in from the DriverContext
```

Comment:
⚠️✋👮 Is this TODO for this PR or for later? If for later, I foresee it will be there for a long time :hehe:

Reply:
Yeah. I feel like I'll either have to do it in the next follow up or, well, it'll wait.

It'll be more consistent if we do it, but we do only ever use the request breaker so it is safe enough as it.